IssueTracker/DataAccess/Models/Comment.cs

```diff
@@ -15,4 +15,5 @@ public class Comment
     public bool IsDeleted { get; set; }
     public User? User { get; set; }
     public IEnumerable<Comment> Replies { get; set; } = Enumerable.Empty<Comment>();
+    public List<FileModel> MetaDatas { get; set; } = new List<FileModel>();//I had to use List
```

Comment:
Use IEnumerable here. The list implements IEnumerable, so you can still use it in LoadCommentDataAsync.

Reply:
but I can't use the add function